iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm

```diff
@@ -84,13 +90,18 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
             _driveThrough = NSLocalizedString(@"drive_through", nil);
           break;
         case MetadataID::FMD_WEBSITE_MENU: _websiteMenu = ToNSString(value); break;
+        case MetadataID::FMD_SELF_SERVICE: _selfService = GetLocalizedMetadataValueString(metaID, value); break;
```

Comment:
@vng how to properly translate these new [type.self_service.no] etc types on both platforms? Isn't this logic repeated on Android?

Reply:
And also I'm a bit lost. There are _types_ type.self_service.no etc., but there is also a _metadata_ field? How does it work? @vng

Reply:
Well ..

We missed it in the previos PR. For unknown reasons, self_service.xxx translations were made like classifier types e.g. type.self_service.xxx in types_strings.txt

So now GetLocalizedMetadataValueString -> GetLocalizedTypeName is used.

Reply:
@kirylkaveryn please add a TODO to review/refactor translations logic later, now it's wrong.